package.json

```diff
@@ -3,26 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.5",
```

Comment:
Do we need this library? Please, check are there any unused dependencies

Reply:
I installed it to try and forgot to remove it. Removed.